### PR TITLE
Fixes a runtime caused by gibs in space

### PR DESCRIPTION
**Pull request by HowToLoLu**
## About The Pull Request
Fixed a runtime due to a signal being unregistered early in a component's lifetime.

Also, despite the PR branch name- I assure you that effects can still drift.

## Why It's Good For The Game
One fix more, one runtime less.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>
Around 150 humans were harmed in the making of this fix.

An additional 21 were harmed to produce this screenshot of the working fix
image

Further evidence of testing can be found at issue #8913

</details>

## Changelog
:cl:
fix: Gibbing in space no longer causes a runtime error
/:cl:

### CODE REVIEW

File: code/datums/components/drift.dm [hunk 1]
```diff
@@ -57,7 +57,7 @@
 	SIGNAL_HANDLER
 	var/atom/movable/movable_parent = parent
 	movable_parent.inertia_moving = FALSE
-	UnregisterSignal(movable_parent, list(COMSIG_MOVABLE_MOVED, COMSIG_MOVABLE_NEWTONIAN_MOVE))
+	UnregisterSignal(movable_parent, (COMSIG_MOVABLE_MOVED))
```
Comment (EvilDragonfiend) [inline code/datums/components/drift.dm]: wrapped signal?

Comment (HowToLoLu) [reply code/datums/components/drift.dm]: Ah- didn't notice I have an extra set of parenthesis leftover; I'll fix that now

**Comment by HowToLoLu** [comment]
Not sure why MapDiffBot2 is upset right now, but it hopefully should end up being fine later-

Worst case: I'll have to figure out what went wrong when I come back next week.

**Comment by HowToLoLu** [comment]
Alright- I think it really is just Github having issues or something- it keeps failing with some kind've configuration error that I think is out of my control.

I also think I screwed up the branch the PR is on- still learning git and how *not* to break it, so I'm closing this and probably redoing this on a better day (or someone else can, it's just one line of code in `components/drift.dm`).

**Pull request closed without merging**